ethereum/history.go

```diff
@@ -129,7 +129,7 @@ func (e *Eth) GetHistory(
 				if err := wormholeAbi.UnpackIntoInterface(&event, "LogMessagePublished", lLog.Data); err != nil {
 					log.Error("error unpacking wormhole abi into interface when querying history", "error", err)
 				}
-				log.Debug("found relevant events during historical query", "block", event.Raw.BlockNumber, "seq", event.Sequence)
+				log.Debug("found relevant events during historical query", "block", lLog.BlockNumber, "seq", event.Sequence)
```

Comment:
Did you test that this actually logs the block number as intended? not very familiar with what is happening here so can't 100% speak to the validity of the change

Reply:
Yes tested!
The logging unintentionally got broken in [this](https://github.com/noble-assets/jester/pull/12) PR.